### ISSUE TITLE
Direct3D 12

### ISSUE THREAD

**Issue by chentiannan** [OGRECave/ogre-next#417]
Is there a D3D12 support program?

**Comment by darksylinc** [comment]
Hi!

There are no current plans for D3D12 support.

**Comment by chentiannan** [comment]
It would be great to support D3D12, hopefully there will be plans for the next release, I want to be a part of it too

**Comment by Alter73** [comment]
Yes, it’s unfortunate that DirectX 12 is still not supported. At this point, it is already quite relevant and not what it was 10 years ago. DirectX 11 is no longer really sufficient for modern needs, and Vulkan does not perform as well on Windows 11. I sincerely hope that D3D12 will eventually be added to the engine.
